### DISCUSSION TITLE
Unable to log into application

### DISCUSSION THREAD

**Discussion by Zxun2** [TEAMMATES/teammates#12471] [Help requests]
Operating System: Windows11
JDK Version: 11.0.17
Node.js Version: v18.5

## Issue 

I am experiencing an error similar to the issues raised [here](https://github.com/TEAMMATES/teammates/issues/11889).
 
These are the steps i have taken **in sequence**: 

1. `./gradlew generateTypes`
1. `npm run start`
1. docker-compose up -d
1. ./gradlew serverRun

**Other attempts:** 

On top of [this](https://github.com/TEAMMATES/teammates/issues/11889#issuecomment-1171676688), 

I have tried to build the frontend with `npm run build` as mentioned [here](https://github.com/TEAMMATES/teammates/issues/11889#issuecomment-1172881358) but i am still getting 404 error on the backend when i try to log in.

![image](https://github.com/TEAMMATES/teammates/assets/63457492/28e767bc-0452-460d-bdf7-dca2b04bd02b)

![image](https://github.com/TEAMMATES/teammates/assets/63457492/bce5e77c-aa40-49b4-9f14-8fa290976128)

Following the guide, 

> Go to any administrator page, e.g /web/admin/home. You may be prompted to log in. You will be granted access only if your account has admin permission as defined in build.properties.

and when I navigate to `/web/admin/home` as mentioned in the [guide](https://teammates.github.io/teammates/development.html), i was given the following error: 
![image](https://github.com/TEAMMATES/teammates/assets/63457492/80567183-cbff-450f-91ba-bf5d59119015)

In the console, 

![image](https://github.com/TEAMMATES/teammates/assets/63457492/a7167808-1122-4953-82f9-228ddea92043)

This is my `build-dev.properties` file,

```
app.enable.devserver.login = true

# When in dev environment, this URL will be whitelisted for CORS and origin checks.
app.frontend.url = http\://localhost:4200

# This is the port to connect with local Datastore emulator.
app.localdatastore.port = 8484

app.taskqueue.active = true
```

Appreciate any help i can get on this!

**Comment by nagarajan-ck** [comment]
Hi
Were you able to resolve this issue, I'm also facing the same when I try to setup.
